app/views/requests/_form.html.erb

```diff
@@ -1,89 +1,57 @@
-          <small class="form-text text-muted">Note: All responsible users will automatically have sudo access rights on the VM.</small>
-        </div>
+        <%= form.text_field :name, label: 'Name *' %>
+        <%= form.text_area :description, rows: 4, label: 'Description *' %>
+      
+        <%# form.collection_select(method, collection, value_method, text_method, options = {}, html_options = {}) %>
+        <%= form.collection_select :project_id, Project.all, :id, :name,
+          options = {selected: form.object.project_id, include_blank: '(none)', label: 'Project *'},
+          html_options = {class: 'selecttwo'} %>
+        <%= form.collection_select :responsible_user_ids, User.all, :id, :human_readable_identifier,
+          options = {selected: @request.responsible_users.collect(&:id),
+                    label: 'Responsible Users *',
+                    help: 'Note: All responsible users will automatically have sudo access rights on the VM.'},
+          html_options = {multiple: true, class: 'selecttwo'} %>
       </div>
-    </div>
+    </div> <%# div class="row" %>
     <div class="row border-bottom mb-4 pb-4">
       <div class="col">
         <h5>Hardware Information</h5>
         You can select a Template or enter custom requirements. If you have special requirements or need an operating
         system that is not in the list you can use the comment field at the bottom of this page.
       </div>
       <div class="col-6">
-        <div class="form-group pb-4 mb-2 border-bottom">
-          <%= form.label "Template" %>
-          <%= form.select :template_id,
-                          options_for_select(@request_templates.map {|request_template|
-                            ["#{request_template.name}: #{request_template.cpu_cores} CPU cores, #{request_template.ram_gb} GB RAM, #{request_template.storage_gb} GB Storage, #{request_template.operating_system}",
-                             "[#{request_template.cpu_cores}, #{request_template.ram_gb}, #{request_template.storage_gb}, '#{request_template.operating_system}']"]}),
-                          {:include_blank => 'None'},
-                          {class: "form-control"} %>
-        </div>
-        <div class="form-group row full-width pt-3">
-          <div class="col">
-            <%= form.label :cpu_cores, "CPU cores *" %>
-            <%= form.number_field :cpu_cores, min: 0, class: 'form-control', id: 'cpu' %>
+        <%= form.select :template_id,
+                        options_for_select(@request_templates.map {|request_template|
+                          ["#{request_template.name}: #{request_template.cpu_cores} CPU cores, #{request_template.ram_gb} GB RAM, #{request_template.storage_gb} GB Storage, #{request_template.operating_system}",
+                          "[#{request_template.cpu_cores}, #{request_template.ram_gb}, #{request_template.storage_gb}, '#{request_template.operating_system}']"]}),
```

Comment:
This formatting could be a method on the `RequestTemplate` model

Reply:
`RequestTemplate#text_summary`. Also uses JSON in the option values.
https://github.com/hpi-swt2/vm-portal/pull/525/commits/e7f21d988e2d5a146d2a66e82e6ff3e20a855e9e